src/components/garagePromotionCard.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, MouseEvent } from "react"
+
+import UploadImagesIcon from "./icons/uploadImage"
+import ArrowRightTeal from "./icons/arrowRight"
+import { FixedCard } from "./card/fixed"
+
+interface Props {
+  dealerName: string
+  dealerLocation: {
+    address?: string
+    zipCode: string
+    city?: string
+  }
```

Comment:
if you go for a more abstract name, I'd also go for more abstract names here. don't focus on the `dealer` so much - this is a specific detail of how the component is currently used. imagine we want to display the same card, but for a blog teaser for example. then it would be more like...

image, title, link, href, footerTitle, footerText (or similar)